src/main/java/com/google/api/generator/engine/ast/StringObjectValue.java

```diff
@@ -43,6 +43,6 @@ public static StringObjectValue withValue(String value) {
 
   @Override
   public String toString() {
-    return value();
+    return String.format("%s%s%s", "\"", value(), "\"");
```

Comment:
Can we do this in `value()`?

Reply:
Sure, I will move the format to value();

Reply:
I would keep this in `toString()` method for two reasons:
1. we need `public abstract String value()` as a property for `setValue()` and override value() would give duplicate methods name.
2.  it might make sense that the value of the string object is just the content of string, and rendering (toString) the object is to add double quote.
WDYT? Thank you.

Reply:
The escaping and full, final string value (including extra quotes) should go into `value()` because it currently represents the raw, generated code values for all Value classes. We don't need to override `value()`, we can simply process the string in `build()`.

Reply:
The escaping and full, final string value (including extra quotes) should go into `value()` because it currently represents the raw, generated code values for all Value classes. We don't need to override `value()`, we can simply process the string in `build()`.

Reply:
The escaping and full, final string value (including extra quotes) should go into value() because it currently represents the raw, generated code values for all Value classes. We don't need to override value(), we can simply process the string in build().

Reply:
sounds cool, investigated for some time, and you are right, the setValue() can be override in build() method. Thanks!